### ISSUE TITLE
Console Updater - Auth Service Issue - Xbox

### ISSUE THREAD

**Issue by MarcusLancaster1** [EDSM-NET/FrontEnd#275]
Hi.

I've been trying the console updater for a few days now... it was working really well up until a week-ish ago... but now all I get is a brief message every minute indicating that

"The Frontier Auth Service is currently not available"

Nothing is being logged... I've tried clearing cache... I've tried reentering my xBox live account info so the consoler updater can connect to Frontier... but nothing is working.

Anyway ideas?

Cheers for now.

Marcus.

**Comment by AnthorNet** [comment]
Can you give me your CMDR name?

**Comment by MarcusLancaster1** [comment]
Hi @AnthorNet 

Yep - "Poppabeans"

Cheers!

**Comment by Deuteronomy1956** [comment]
I have a similar issue.  I had it working for about a day (last week) but the next day I got a message that told me I needed to unlink and then relink my Xbox.  I unlinked but then  it wouldn't let me link again.  I have a pc account but mainly only play on Xbox.  My CMDR name is Deuteronomy1956 on both accounts.  I have two different email accounts but I would prefer to use the same for both.  Is that possible?  Anyway, when I tried to relink it told me neither emails were available.  So I tried to delete my pc account and I think I really screwed it up.  Am I in the right place for help?  Thanks.
